_docs/setup/kubernetes/helm-install.md

```diff
@@ -48,31 +80,35 @@ may be used in the same Helm operation.
 following table:
 
-| Helm Variable                | Possible Values    | Default Value              | Purpose of Key                                 |
-|------------------------------|--------------------|----------------------------|------------------------------------------------|
-| global.namespace             | any Kubernetes ns  | istio-system               | Specifies the namespace for Istio              |
-| global.initializer.enabled   | true/false         | true                       | Specifies whether to use the Initializer       |
-| global.proxy.hub             | registry+namespace | release registry/namespace | Specifies the HUB for the proxy image          |
-| global.proxy.tag             | image tag          | release unique hash        | Specifies the TAG for the proxy image          |
-| global.proxy.debug           | true/false         | false                      | Specifies whether proxy is run in debug mode   |
-| global.pilot.hub             | registry+namespace | release registry/namespace | Specifies the HUB for the pilot image          |
-| global.pilot.tag             | image tag          | release unique hash        | Specifies the TAG for the pilot image          |
-| global.pilot.enabled         | true/false         | true                       | Specifies whether pilot is enabled/disabled    |
-| global.security.hub          | registry+namespace | release registry/namespace | Specifies the HUB for the ca image             |
-| global.security.tag          | image tag          | release unique hash        | Specifies the TAG for the ca image             |
-| global.security.enabled      | true/false         | false                      | Specifies whether security is enabled/disabled |
-| global.mixer.hub             | registry+namespace | release registry/namespace | Specifies the HUB for the mixer image          |
-| global.mixer.tag             | image tag          | release unique hash        | Specifies the TAG for the mixer image          |
-| global.mixer.enabled         | true/false         | true                       | Specifies whether mixer is enabled/disabled    |
-| global.hyperkube.hub         | registry+namespace | quay.io/coreos/hyperkube   | Specifies the HUB for the hyperkube image      |
-| global.hyperkube.tag         | image tag          | v1.7.6_coreos.0            | Specifies the TAG for the hyperkube image      |
-| global.ingress.use_nodeport  | true/false         | false                      | Specifies whether to use nodeport or LB        |
-| global.ingress.nodeport_port | 32000-32767        | 32000                      | If nodeport is used, specifies its port        |
-
-## Uninstalling
-
-* Uninstall Istio:
-
-  ```bash
-  helm delete --purge istio
-  ```
+| Parameter | Description | Values | Default |
+| --- | --- | --- | --- |
+| `global.hub` | Specifies the HUB for most images used by Istio | registry/namespace | `docker.io/istionightly` |
```

Comment:
istionightly should be not be the default...

Reply:
Only documenting current state of master.  The make system generates yamls based upon $(HUB) and $(TAG) as part of the release process.  I don't know precisely what these tags should be for defaults.  0.8.0?